components/admin_console/billing/billing_summary/billing_summary.tsx

```diff
@@ -202,62 +239,64 @@ export const lastInvoiceInfo = (invoice: any, product: any, fullCharges: any, pa
                     </div>
                 </div>
             ))}
-            {partialCharges.length &&
-            <div className='BillingSummary__lastInvoice-partialCharges'>
-                <FormattedMessage
-                    id='admin.billing.subscriptions.billing_summary.lastInvoice.partialCharges'
-                    defaultMessage='Partial charges'
-                />
-                <OverlayTrigger
-                    delayShow={500}
-                    placement='bottom'
-                    overlay={(
-                        <Tooltip
-                            id='BillingSubscriptions__seatOverageTooltip'
-                            className='BillingSubscriptions__tooltip BillingSubscriptions__tooltip-right'
-                            positionLeft={390}
+            {partialCharges.length ?
```

Comment:
Good catch! Could we do `Boolean(partialCharges.length) && ...` instead of `{partialCharges.length ? ... : null`. It's a lot of lines to scan to see that what we're doing is not making two large conditional render branches but safeguarding against displaying `0`.